docker/config/requirements.txt

```diff
@@ -1,117 +1,117 @@
 # Replacement for the local package installs from CodeArtifact
-alembic==1.5.8
-amqp==2.6.1
-apache-airflow==2.0.2
-apache-airflow-providers-amazon==1.3.0
-apache-airflow-providers-celery==1.0.1
-apache-airflow-providers-ftp==1.0.1
-apache-airflow-providers-http==1.1.1
-apache-airflow-providers-imap==1.0.1
-apache-airflow-providers-sqlite==1.0.2
+alembic==1.7.4
+amqp==5.0.6
+apache-airflow==2.2.2
+apache-airflow-providers-amazon==2.4.0
+apache-airflow-providers-celery==2.1.0
+apache-airflow-providers-ftp==2.0.1
+apache-airflow-providers-http==2.0.1
+apache-airflow-providers-imap==2.0.1
+apache-airflow-providers-sqlite==2.0.1
 apispec==3.3.2
 argcomplete==1.12.3
 attrs==20.3.0
 babel==2.9.1
 billiard==3.6.4.0
 blinker==1.4
-boto3==1.17.53
-botocore==1.20.53
+boto3==1.18.65
+botocore==1.21.65
 cached-property==1.5.2
 cattrs==1.5.0
-celery==4.4.7
+celery==5.2.0
 certifi==2020.12.5
-cffi==1.14.5
+cffi==1.15.0
 chardet==3.0.4
 click==7.1.2
 clickclick==20.10.2
 colorama==0.4.4
-colorlog==5.0.1
+colorlog==4.8.0
 commonmark==0.9.1
 connexion==2.7.0
-croniter==0.3.37
-cryptography==3.4.7
+croniter==1.0.15
+cryptography==3.4.8
 defusedxml==0.7.1
-dill==0.3.2
-dnspython==1.16.0
+dill==0.3.1.1
+dnspython==2.1.0
 docutils==0.16
-email-validator==1.1.2
+email-validator==1.1.3
 Flask==1.1.2
-Flask-AppBuilder==3.3.0
-Flask-Babel==1.0.0
+Flask-AppBuilder==3.3.4
+Flask-Babel==2.0.0
 Flask-Caching==1.10.1
 Flask-JWT-Extended==3.25.1
 Flask-Login==0.4.1
-Flask-OpenID==1.2.5
+Flask-OpenID==1.3.0
 Flask-SQLAlchemy==2.5.1
 Flask-WTF==0.14.3
-flower==0.9.7
+flower==1.0.0
 graphviz==0.16
 gunicorn==19.10.0
```

Comment:
I get `ERROR: Cannot install apache-airflow==2.2.2 and gunicorn==19.10.0 because these package versions have conflicting dependencies.` and some other similar ones for some other dependencies. Any ideas why this might happen?